client/rpc-core/txpool/Cargo.toml

```diff
@@ -16,4 +16,4 @@ jsonrpc-derive = "14.0.3"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 
-fc-rpc-core = { git = "https://github.com/purestake/frontier", branch = "notlesh-moonbeam-v0.7" }
+fc-rpc-core = { git = "https://github.com/purestake/frontier", branch = "update-substrate-bn-128" }
```

Comment:
Wasn't this merged into master?

Reply:
not that I know but I can make a PR only with that update if thats better

Reply:
this change is in master now, so this discussion is no longer relevant. I've updated the pr and its title to reflect that.